source/includes/cloudstack/_snapshots.md

```diff
@@ -88,3 +88,47 @@ Attributes | &nbsp;
 `volumeId`<br/>*UUID* | The id of the [volume](#cloudstack-volumes) that was snapshotted
 `volume`<br/>*string* | The name of the [volume](#cloudstack-volumes) that was snapshotted
 `volumeType`<br/>*string* | The type of the [volume](#cloudstack-volumes) that was snapshotted
+
+
+<!-------------------- CREATE A SNAPSHOT OF A VOLUME -------------------->
+
+
+#### Create a snapshot
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/compute-on/testing/snapshots?operation=create"
+
+# Request should look like this
+```
+```json
+{
+  "volumeId": "4fe54594-a788-442c-b7a8-0237f7a4f70d",
+  "name": "LionelMessi",
+  "rapid": true
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/snapshots?operation=create</code>
+
+Create a snapshot of an existing storage [volume](#cloudstack-storage). Note that the volume must be attached to an instance. A detached volume cannot be **snapshot*****'ed***. You can get a list of volumes using [this](#cloudstack-list-volumes) API.
```

Comment:
You can remove `storage` from the first sentence, it's implicit. Also, a volume only needs to be in the `Ready` state for the **Take snapshot** operation. It doesn't have to be attached to an instance (it must have been at some point to become _Ready_, but it can be detached).

Reply:
Removed `storage`.

Reply:
Yes, it only needs to be in `READY` state. It had failed for me _(when I checked on a detached volumne)_ because the beta2 instance was overloaded and I assumed it without looking at the code. Fixed it